Riot/Assets/pr.lproj/Vector.strings

```diff
@@ -1,640 +1,632 @@
-/*

```

Comment:
Note: I’m not exactly sure why this happens to happen when ran the script to clean the strings, but Pirate is an excluded language from the project anyway, so personally I’m not convinced there’s too much to be worries about.